tests/vm_state_test.go

```diff
@@ -175,10 +175,10 @@ var _ = Describe("[sig-storage]VM state", decorators.SigStorage, decorators.Requ
 			err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Delete(context.Background(), vm.Name, &k8smetav1.DeleteOptions{})
 			Expect(err).ToNot(HaveOccurred())
 		},
-			Entry("TPM across migration and restart", true, false, "migrate", "restart"),
-			Entry("TPM across restart and migration", true, false, "restart", "migrate"),
-			Entry("EFI across migration and restart", false, true, "migrate", "restart"),
-			Entry("TPM+EFI across migration and restart", true, true, "migrate", "restart"),
+		        Entry("[test_id:10818]TPM across migration and restart", true, false, "migrate", "restart"),
```

Comment:
I think the whitespace change here is causing the `pull-kubevirt-generate` presubmit lane to fail every time on this PR. 

https://prow.ci.kubevirt.io/pr-history/?org=kubevirt&repo=kubevirt&pr=11157